Added, Weed lab,Skunk bagging,missing images

Andyauk opened the pull request:
Tested and using the latest copy of ps-drugprocessing

Added, Weed lab.
Added, Skunk bagging
and images for missing Baking Soda and Trimming Scissors

RLEpixs reviewed (commented):
Finally some Weed Lab, thanks <2

Iitsmemitchell commented:
works for me!

nzkfc commented:
@RLEpixs I added a few new drugs and labs months ago, but noticed on your one you have "CWarehouse = true" in your local, not sure if this has to be unique e.g. "WWarehouse = true"? (just asking for the sake of the PR rather then getting it working.)

Andyauk commented:
> @RLEpixs I added a few new drugs and labs months ago, but noticed on your one you have "CWarehouse = true" in your local, not sure if this has to be unique e.g. "WWarehouse = true"? (just asking for the sake of the PR rather then getting it working.)

CWarehouse should be WWearhouse in the weed lab section, good point out I'll change that in a bit. Funny as not had any issues for months of using it. When I added to use of the weed lab I used the code from coke as it's what meth uses too.

nzkfc commented:
Yeah there's a few oddities in drug processing, but it works so it's fine
:)

On Sat, 4 Mar 2023, 10:35 pm Andyauk, ***@***.***> wrote:

> @RLEpixs <https://github.com/RLEpixs> I added a few new drugs and labs
> months ago, but noticed on your one you have "CWarehouse = true" in your
> local, not sure if this has to be unique e.g. "WWarehouse = true"? (just
> asking for the sake of the PR rather then getting it working.)
>
> CWarehouse should be WWearhouse in the weed lab section, good point out
> I'll change that in a bit. Funny as not had any issues for months of using
> it. When I added to use of the weed lab I used the code from coke as it's
> what meth uses too.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Project-Sloth/ps-drugprocessing/pull/111#issuecomment-1454678503>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABGHUHPGPIAUIVZKNSFIFHLW2MEG3ANCNFSM6AAAAAAUV5YBQE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

Andyauk commented:
> Yeah there's a few oddities in drug processing, but it works so it's fine :)
> […](#)
> On Sat, 4 Mar 2023, 10:35 pm Andyauk, ***@***.***> wrote: @RLEpixs <https://github.com/RLEpixs> I added a few new drugs and labs months ago, but noticed on your one you have "CWarehouse = true" in your local, not sure if this has to be unique e.g. "WWarehouse = true"? (just asking for the sake of the PR rather then getting it working.) CWarehouse should be WWearhouse in the weed lab section, good point out I'll change that in a bit. Funny as not had any issues for months of using it. When I added to use of the weed lab I used the code from coke as it's what meth uses too. — Reply to this email directly, view it on GitHub <[#111 (comment)](https://github.com/Project-Sloth/ps-drugprocessing/pull/111#issuecomment-1454678503)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABGHUHPGPIAUIVZKNSFIFHLW2MEG3ANCNFSM6AAAAAAUV5YBQE> . You are receiving this because you commented.Message ID: ***@***.***>

tbh i only used what was already there, but yes it works fine.

MonkeyWhisper commented:
Hi everyone,

I just been keeping an eye on this PR making sure no one reported anything.

This script I have been dragging since the QBUS days and it has evolved so so much since really... I still haven't found a proper drug script

Thanks for the PR
